builds/misc/templates/build-packages.yaml

```diff
@@ -138,4 +138,220 @@ stages:
             inputs:
               PathtoPublish: '$(build.artifactstagingdirectory)'
               ArtifactName: 'iotedged-$(os)-$(arch)'
-            condition: and(succeededOrFailed(), or(eq(${{ parameters['E2EBuild'] }}, false), eq(variables.arch,'amd64')))
\ No newline at end of file
+            condition: and(succeededOrFailed(), or(eq(${{ parameters['E2EBuild'] }}, false), eq(variables.arch,'amd64')))
+
+      ################################################################################
+      - job: mariner_linux
+      ################################################################################
+        displayName: Mariner_Linux
+        pool:
+          # We are using Linux image that lives in Windows pool. The Windows pool by default uses Standard_D4s_v3 SKU.
+          # EFLOW requires ~85GB to build the edgelet artifacts.
+          name: $(pool.windows.name)
+          demands:
+          - ImageOverride -equals agent-aziotedge-ubuntu-20.04-docker-large-disk
+        variables:
+          NugetSecurityAnalysisWarningLevel: warn
+        strategy:
+          matrix:
+            CBL-Mariner1.0-amd64:
+              arch: amd64
+              os: mariner
+              os_version: 1
+              mariner_release: 1.0-stable
+              target.iotedged: builds/mariner1/out/RPMS
+              target.logs: builds/mariner1/build/logs/pkggen/rpmbuilding/
+              IISCommit: 158a726f0684cdfa5609191c2c9ace0123530522
+            CBL-Mariner2.0-amd64:
+              arch: amd64
+              os: mariner
+              os_version: 2
+              mariner_release: 2.0-stable
+              target.iotedged: builds/mariner2/out/RPMS
+              target.logs: builds/mariner2/build/logs/pkggen/rpmbuilding/
+              IISCommit: 158a726f0684cdfa5609191c2c9ace0123530522
```

Comment:
We should have a more dynamic way to do this. I'm ok with the initial PR to go in with hard-coded value, but we should address it once the 1.3.0 is cut.

Reply:
I can remove this once we have a azure-identity-service build that has the mariner changes. I know that effected release/1.2 but i am not sure about their master builds.